doc/source/security/using-kmip.rst

```diff
@@ -0,0 +1,78 @@
+
+* Streamline encryption key management
+
+* Eliminate redundant key management processes
+
+Component installation
+--------------------------------------
+
+The KMIP component must be installed with a manifest. A keyring component is not loaded with the ``--early-plugin-load`` option on the server. The server uses a manifest and the component consults its configuration file during initialization. You should only load a keyring component with a manifest file. Do not use the ``INSTALL_COMPONENT`` statement, which loads the keyring components too late in the startup sequence of the server. For example, ``InnoDB`` requires the component, but because the components are registered in the ``mysql.component`` table, this table is loaded after ``InnoDB`` initialization. 
+
+You should create a global manifest file named ``mysqld.my`` in the installation directory and, optionally, create a local manifest file, also named ``mysqld.my`` in a data directory.
+
+To install a keyring component, you must do the following:
+
+1. Write a manifest in a valid JSON format
+2. Write a configuration file
+
+A manifest file indicates which component to load. During startup, the server can read either a global manifest file, or a global manifest file that points to a local manifest file.
+
+The server reads the global manifest file from the installation directory. If the global manifest file points to a local manifest file, the server also reads the local manifest file in the data directory. If the manifest file does not exist, the server does not load the component associated with that file.
+
+A local manifest file allow loading different components for multiple instances of the server. The loading instructions are specific to that data directory. Different server instances can use different keyring components.
+
+.. note:: 
+
+    Enable only one keyring plugin or one keyring component at a time for each server instance. Enabling multiple keyring plugins or keyring components or mixing keyring plugins or keyring components is not supported and may result in data loss.
+
+The following is an example of a global manifest file that does not use local manifests:
+
+.. code-block:: json
+
+    {
+     "read_local_manifest": false,
+     "components": "file:///component_keyring_kmip"
+    }
+
+The following is an example of a global manifest file that points to a local manifest file:
+
+.. code-block:: json
+
+    {
+     "read_local_manifest": true
+    }
+
+The following is an example of a local manifest file:
+
+.. code-block:: json
+
+    {
+     "components": "file:///component_keyring_kmip"
+    }
+
+Configure the component by reviewing the component's configuration instructions. The following **JSON** example could be an example either the global configuration file or a local configuration file for the KMIP component. Both files contain the same values.
```

Comment:
The following **JSON** example could be an example OF either

Reply:
Rewritten